src/backend/shortcuts/shortcuts/shortcuts.ts

```diff
@@ -160,7 +163,7 @@ function shortcutFiles(gameTitle: string) {
     }
     case 'darwin':
       menuFile = join(userHome, 'Applications', `${gameTitle}.app`)
-      desktopFile = join(userHome, 'Applications', `${gameTitle}.app`)
+      desktopFile = menuFile
```

Comment:
just to make it more obvious it's the same path (I actually wanted to return `null` here instead since we don't add desktop shortcuts, but it was getting annoying with the types in other places if this is potentially null)